projects/plugins/boost/app/assets/src/js/features/premium-tooltip/premium-tooltip.module.scss

```diff
@@ -27,3 +27,18 @@
 	bottom: 0;
 	fill: #8c8f94;
 }
+
+.upgrade-cta {
+	p {
+		display: block;
+	}
+
+	:global(button) {
+		margin-top: 0;
+		margin-bottom: 0;
+	}
```

Comment:
Something seems off with how we need to reference buttons globally.

Reply:
Styling an element within another component. I dunno how to do it otherwise.

Reply:
That's something I think we need to think about 🧠